docs/examples/Ch-ExamplesAddingModbusDevice.rst

```diff
@@ -2,459 +2,273 @@
 
+To add a device after starting the services, complete the following steps:
 
-In this example the name is "TemperatureDegF".
+1. Upload the device profile above to metadata with a POST to http://localhost:48081/api/v1/deviceprofile/uploadfile and add the file as key “file” to the body in form-data format, and the created ID will be returned.  The following figure is an example if you use Postman to send the request
 
-**Create addressable**
+    .. image:: upload_profile.png
+        :scale: 50%
+        :alt: Uploading the Profile
 
-In this section you will need to supply a path the the item you want to schedule.
+2. Add the addressable containing reachability information for the device with a POST to http://localhost:48081/api/v1/addressable: 
 
-The path outline is:
+    a. If IP connected, the body will look something like: { “name”: “Motor”, “method”: “GET”, “protocol”: “HTTP”, “address”: “10.0.1.29”, “port”: 502 } 
+    b. If serially connected, the body will look something like: { “name”: “Motor”, “method”: “GET”, “protocol”: “OTHER”, “address”: “/dev/ttyS5,9600,8,1,1”, “port”: 0 } (address field contains port, baud rate, number of data bits, stop bits, and parity bits in CSV form)
 
-/api/v1/device/{device id}/{command name}
+3. Ensure the Modbus device service is running, adjust the service name below to match if necessary or if using other device services.
 
-In this case, the address would be
+4. Add the device with a POST to http://localhost:48081/api/v1/device, the body will look something like::
 
-::
+    {
+      "description": "MicroMax Variable Speed Motor",
+      "name": "Variable Speed motor",
+      "adminState": "unlocked",
+      "operatingState": "enabled",
+      "addressable": {
+        "name": "Motor"
  
-  /api/v1/device/5a1dd585e4b0c3936013123d/TemperatureDegF
+      },
+      "labels": [
+ 
+      ],
+      "location": null,
+      "service": {
+        "name": "edgex-device-modbus"
+ 
+      },
+      "profile": {
+        "name": "GS1-VariableSpeedMotor"
+ 
+      }
+    }
 
-  /POST addressable
+   The addressable name must match/refer to the addressable added in Step 2, the service name must match/refer to the target device service, and the profile name must match the device profile name from Step 1.
 
-      “name”: “schedule-mbus-rth-lcd”
+Execute Commands
+================
 
-      “protocol”: “HTTP”
+Now we're ready to run some commands.
 
-      “address”: “edgex-device-modbus”
+Find Executable Commands
+------------------------
 
-      “port”: “49991”
+Use the following query to find executable commands::
 
-      “path”: “/api/v1/device/5a1dd585e4b0c3936013123d/TemperatureDegF”
- 
-      “method”: “GET”  *** This will need to be added ***
+    photon-ip:48082/api/v1/device
+
+    .. image:: commands.png
```

Comment:
This image isn't being formatted right, I suspect the line above it has a space that's breaking the sphinx rendering

Reply:
Updated